tls/s2n_security_policies.c

```diff
@@ -1062,6 +1075,7 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version = "default_tls13", .security_policy = &security_policy_default_tls13, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "default_fips", .security_policy = &security_policy_default_fips, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "20230317", .security_policy = &security_policy_20230317, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
+    { .version = "20230317_12", .security_policy = &security_policy_20230317_12, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
```

Comment:
I think I'd prefer to stick with a normal "date" name instead of the `_12` suffix, and then just add a comment describing that this is effectively the same as `20230317` but without TLS 1.3

Reply:
Fair. I updated it to yesterday's date to avoid an April 1st policy :)